### ISSUE TITLE
Consider changing .go extension to .gameobject

### ISSUE THREAD

**Issue by Malmer** [defold/editor2-issues#862]
Because external editors and tools mistake the files for Golang code files. Also it would be more consistent with the naming of the other things, such as .collection, .particlefx and .material.

**Comment by Malmer** [comment]
Obviously .go should still be supported for legacy reasons, but new files should always be with the new longer extension.

**Comment by subsoap** [comment]
While we are at it change game.project to something unique too. If .defold is used as the filename then project main filenames can have any name not just be "game"

mygame.defold
blossom.defold

**Comment by Malmer** [comment]
Yes, that would make sense. Having files named anything as generic as ".project" associated with Defold is probably a bit of a stretch for many users. I don't mind it being called game.defold though if that is a requirement for things to work smoothly.

**Comment by mosmetro** [comment]
This is great proposal! Must be done sooner or later. Better sooner. Seeing Ben James repository in Go trending (several times!) is ridiculous!
